src/main/scala/nl/knaw/dans/easy/pid/ServiceStarter.scala

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.pid
+
+import java.io.File
+
+import com.typesafe.config.ConfigFactory
+import nl.knaw.dans.easy.pid.microservice.HazelcastService
+import org.apache.commons.daemon.{Daemon, DaemonContext}
+import org.eclipse.jetty.server.Server
+import org.slf4j.LoggerFactory
+
+class ServiceStarter extends Daemon {
```

Comment:
I got this in my branch as well, in a slightly different way but it looks quite the same! Accepted for now, I'll merge it with my branch and see if I need to add some things...